From: KevinDuan <duankaiwen@kylinos.cn>
Date: Tue, 18 Aug 2026 08:59:28 +0000
Subject: Translated using Weblate (Spanish)

Currently translated at 100.0% (10 of 10 strings)

Translation: openkylin-huanghe/qt5-ukui-platformtheme-qt5
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/qt5-ukui-platformtheme-qt5/es/
---
 .../translations/qt5-ukui-platformtheme_es.ts        | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/qt5-ukui-platformtheme/translations/qt5-ukui-platformtheme_es.ts b/qt5-ukui-platformtheme/translations/qt5-ukui-platformtheme_es.ts
index 910f15c..d222bd5 100644
--- a/qt5-ukui-platformtheme/translations/qt5-ukui-platformtheme_es.ts
+++ b/qt5-ukui-platformtheme/translations/qt5-ukui-platformtheme_es.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../widget/messagebox/message-box.cpp" line="97"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Cerrar</translation>
     </message>
 </context>
 <context>
@@ -14,12 +14,12 @@
     <message>
         <location filename="../widget/messagebox/message-box.h" line="272"/>
         <source>Executable &apos;%1&apos; requires Qt %2, found Qt %3.</source>
-        <translation type="unfinished"></translation>
+        <translation>El archivo ejecutable &quot;%1&quot; requiere la cantidad %2, se encontró la cantidad %3.</translation>
     </message>
     <message>
         <location filename="../widget/messagebox/message-box.h" line="274"/>
         <source>Incompatible Qt Library Error</source>
-        <translation type="unfinished"></translation>
+        <translation>Error de biblioteca Qt incompatible</translation>
     </message>
 </context>
 <context>
@@ -28,7 +28,7 @@
         <location filename="../widget/messagebox/message-box.cpp" line="362"/>
         <location filename="../widget/messagebox/message-box.cpp" line="1090"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>Confirmar</translation>
     </message>
 </context>
 <context>
@@ -38,13 +38,13 @@
         <location filename="../widget/messagebox/message-box.cpp" line="881"/>
         <location filename="../widget/messagebox/message-box.cpp" line="1517"/>
         <source>Show Details...</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar detalles...</translation>
     </message>
     <message>
         <location filename="../widget/messagebox/message-box.cpp" line="881"/>
         <location filename="../widget/messagebox/message-box.cpp" line="1517"/>
         <source>Hide Details...</source>
-        <translation type="unfinished"></translation>
+        <translation>Ocultar detalles...</translation>
     </message>
 </context>
 <context>
@@ -79,7 +79,7 @@
     </message>
     <message>
         <source>Use current sorting for all folders</source>
-        <translation type="unfinished"></translation>
+        <translation>Todos los directorios usan el orden actual</translation>
     </message>
     <message>
         <source>List View</source>
@@ -92,7 +92,7 @@
     <message>
         <location filename="../widget/messagebox/message-box.cpp" line="730"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Cerrar</translation>
     </message>
 </context>
 <context>
@@ -100,12 +100,12 @@
     <message>
         <location filename="../../libqt5-ukui-style/animations/tabwidget/ukui-tabwidget-default-slide-animator-factory.h" line="50"/>
         <source>Default Slide</source>
-        <translation type="unfinished"></translation>
+        <translation>Diapositiva predeterminada</translation>
     </message>
     <message>
         <location filename="../../libqt5-ukui-style/animations/tabwidget/ukui-tabwidget-default-slide-animator-factory.h" line="51"/>
         <source>Let tab widget switch with a slide animation.</source>
-        <translation type="unfinished"></translation>
+        <translation>Hacer que el widget de pestañas cambie con una animación de diapositiva.</translation>
     </message>
 </context>
 <context>
